### DISCUSSION TITLE
/cmdVel topic: cmdVel(roll, pitch, yawrate, thrust)

### DISCUSSION THREAD

**Discussion by ntukenmez3** [bitcraze/discussions#1008] [Q&A]
We'd like to access `/cmdVel` topic to send the related control inputs (i.e., roll, pitch, yawrate, thrust) to Crazyflie. However, when we start Crazyflie, we could not see the `/cmdVel` topic (we assumed this is a default topic for Crazyflie). I added the topic list below. How could we fix it, let us now if you have any useful explanation for this situation, thanks.

```
ntukenmez3@ae-icps-407120:~/Documents/2023-crazy-flie-1/crazyflie_ros2_experimental/crazyflie_ros2_implementation$ ros2 topic list 
/cmd_vel
/odom
/parameter_events
/rosout
/scan
/tf
/zrange
```

**Comment by knmcguire** [comment]
Hi! We don't have anything in our api's that gives out a /cmdvel topic or anything ROS related.

 This is a question for the crazyswarm2 discussion list: https://github.com/IMRCLab/crazyswarm2/discussions

**Reply by knmcguire** [reply]
ah wait.... this is my experimental github repo. I see you also asked this question here: https://robotics.stackexchange.com/questions/105081/cmdvel-topic-cmdvelroll-pitch-yawrate-thrust-for-crazyflie.

Let's talk there

**Reply by ntukenmez3** [reply]
Yes, that repository is yours.
